integration-tests/environments/react-native/harness/runner.js

```diff
@@ -16,162 +16,212 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-const { MochaRemoteServer } = require("mocha-remote-server");
-const { timeout, TimeoutError } = require("promise-timeout");
+const {MochaRemoteServer} = require('mocha-remote-server');
+const {timeout, TimeoutError} = require('promise-timeout');
+const yargs = require('yargs');
+const path = require('path');
 
-const rn = require("./react-native-cli");
-const android = require("./android-cli");
-const xcode = require("./xcode-cli");
+const rn = require('./react-native-cli');
```

Comment:
not to be picky but why we need this change :)

Reply:
With the way the package is currently setup, it is [extending the react native eslint config](https://github.com/realm/realm-js/blob/master/integration-tests/environments/react-native/.eslintrc.js#L3), which [uses single-quotes for strings](https://github.com/facebook/react-native/blob/master/packages/eslint-config-react-native-community/index.js#L261): We could change this to extend from our eslintrc (located in the root of the repository), to keep a consistent style across all our code.

This change was introduced simply because my editor is setup to format on save in compliance with the configuration in the package folder.